blockchain/block.go

```diff
@@ -8,6 +8,15 @@ import (
 	"io"
 )
 
+const (
```

Comment:
I'll just say I didn't add these variables on this PR - I just added `MaxTransactionsPerBlock`, not sure why it's all green 🤔. And I totally agree about the constant size.